content/src/service/validations/validator.ts

```diff
@@ -0,0 +1,31 @@
+import { createValidator as validator, ExternalCalls, Validator as IValidatorComponent } from '@dcl/content-validator'
+import { Authenticator } from 'dcl-crypto'
+import { EnvironmentConfig } from '../../Environment'
+import { AppComponents } from '../../types'
+
+export function createValidator(
+  components: Pick<AppComponents, 'storage' | 'catalystFetcher' | 'authenticator' | 'env'>
+): IValidatorComponent {
+  const externalCalls: ExternalCalls = {
+    isContentStoredAlready: (hashes) => components.storage.exist(hashes),
+    fetchContentFileSize: (hash) => components.storage.size(hash),
+    ownerAddress: (auditInfo) => Authenticator.ownerAddress(auditInfo.authChain),
```

Comment:
what does this do? Checks if the signature is correct or gets the address from an authchain?

Reply:
gets the address used in multiple validations like access checkers, these external calls don't validate anything but provide information to the validator

there is another validation for the signature in `@dcl/content-validator`